components/pages/WebsiteDetails.js

```diff
@@ -170,8 +165,8 @@ export default function WebsiteDetails({ websiteId }) {
           <DetailsComponent
             {...tableProps}
             height={500}
-            limit={false}
```

Comment:
This is probably the reason for this: https://github.com/Hashnode/umami/pull/4#discussion_r1530577660

Reply:
This would be the case for umami which loads all data at once
We wanted to paginate the list hence we need this limit i.e. how many items to load in a batch.
We could make it a relatively higher number here.